src/ray/gcs/tables.h

```diff
@@ -208,6 +209,27 @@ class Log : public LogInterface<ID, Data>, virtual public PubsubInterface<ID> {
     static std::hash<ray::UniqueID> index;
     return shard_contexts_[index(id) % shard_contexts_.size()];
   }
+
+  /// Subscribe to any modifications to the key. The caller may choose
+  /// to subscribe to all modifications, or to subscribe only to keys that it
+  /// requests notifications for. This may only be called once per Log
+  /// instance.
```

Comment:
Can you specify how this method is different from `Log::Subscribe`?

Reply:
Added more detailed comment in code. Also paste it here:

> This function is different from public version due to an additional parameter mode in NotificationCallback. Therefore this supports notifications of remove operations.